CommandMaster/script.json

```diff
@@ -2,7 +2,7 @@
     "$schema": "https://github.com/DameryDad/roll20-api-scripts/blob/CommandMasterAPI/CommandMaster/Script.json",
     "name": "CommandMaster",
     "script": "CommandMaster.js",
-    "version": "1.4.05",
+    "version": "1.4.06",
     "previousversions": ["1.020","1.022","1.024","1.025","2.027","1.3.00","1.3.01","1.3.02","1.3.03","1.3.04","1.4.01","1.4.02","1.4.04"],
```

Comment:
You should make sure to include the previous version of the script in this array

Reply:
As per Magic Master, I don't want users loading v1.4.05 as it has bugs fixed in v1.4.06.  I've skipped versions before, without objections.  What's changed?  This needs to go for RPGMaster suite to be at v1.4.06 overall.  Existing users might experience problems (and will certainly still have the bugs in v1.4.05) if this is not released today